mbed_host_tests/host_tests_conn_proxy/conn_primitive_serial.py

```diff
@@ -101,11 +103,11 @@ def write(self, payload, log=False):
                 self.serial.write(payload)
                 if log:
                     self.logger.prn_txd(payload)
+                return payload
         except SerialException as e:
             self.serial = None
             self.LAST_ERROR = "connection lost, serial.write(%d bytes): %s"% (len(payload), str(e))
             self.logger.prn_err(str(e))
-        return payload
```

Comment:
In Python in absence of ```return``` statement LHS value will get a ```None```. Lets not rely on this implicit behaviour. It is less readable. and don't know if it may change!
Perhaps in exception block ```payload=None``` can be done leaving rest of the function intact.